service/frontend/nexus_handler.go

```diff
@@ -273,6 +275,37 @@ func (c *operationContext) shouldForwardRequest(ctx context.Context, header nexu
 		c.forwardingEnabledForNamespace(c.namespaceName)
 }
 
+// enrichNexusOperationMetrics enhances metrics with additional Nexus operation context based on configuration.
+func (c *operationContext) enrichNexusOperationMetrics(service, operation string, requestHeader map[string]string) {
+	conf := c.metricTagConfig(c.namespaceName)
+	if conf == nil {
+		return
+	}
+
+	var tags []metrics.Tag
+
+	if conf.IncludeServiceTag {
+		tags = append(tags, metrics.NexusServiceTag(service))
+	}
+
+	if conf.IncludeOperationTag {
+		tags = append(tags, metrics.NexusOperationTag(operation))
+	}
+
+	for _, mapping := range conf.HeaderTagMappings {
+		value, ok := requestHeader[mapping.SourceHeader]
```

Comment:
`Get` is case insensitive.

```suggestion
		value, ok := requestHeader.Get(mapping.SourceHeader)
```